.github/workflows/release-post-merge.yml

```diff
@@ -0,0 +1,73 @@
+# GitHub workflow for createing release.
+# Trigger release branch should be merge into main
+
+name: Create Release.
+
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  create-release:
+    name: Create Release
+    if: (github.event.pull_request.merged == true) && (startsWith(github.event.pull_request.head.ref, 'release'))
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Print Env and Get version
+      id: tag
+      env:
+        BRANCH: ${{ github.event.pull_request.head.ref }}
+      run: |
+        echo "BRANCH:$BRANCH"
+        version=$(echo $BRANCH | awk -F '/' '{print $2}')
+        tag="v${version}"
+        echo "::set-output name=version::$version"
+        echo "::set-output name=tag::$tag"
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0 #needs for tags
+
+    - name: Create configuration package
+      run: |
+        set -x
+        tar czvf ao-all-in-one-${{ steps.tag.outputs.version }}.tar.gz -C deploy all-in-one-${{ steps.tag.outputs.version }}.yaml
```

Comment:
can it be `tar czvf atlas-operator-all-in-one-${{ steps.tag.outputs.version }}.tar.gz -C deploy all-in-one.yaml` instead?